utility/include/isobus/utility/to_string.hpp

```diff
@@ -14,6 +14,7 @@
 
 #include <sstream>
 #include <string>
+#include "../../isobus/include/isobus/isobus/can_constants.hpp"
```

Comment:
Since this is a library dependency of the main library, probably best to not cross these streams.

`object_id_to_string` is literally only used in the VT client right now anyways, so might just want to move the changes from this file to that file for now

Reply:
You are right, moved this to isobus_virtual_terminal_objects.hpp as it fits there better.